From: Jiri Kosina <jkosina@suse.cz>
Date: Thu, 14 Sep 2017 14:15:36 -0700
Subject: livepatch: __kgr_shadow_get_or_alloc() is local to shadow.c
Git-commit: 5d9da759f7587c87252ef98e70bc0b4a89e4d036
Patch-mainline: v4.15-rc1
References: bsc#1082299 fate#313296

... therefore make it static.

Fixes: 439e7271dc2 ("livepatch: introduce shadow variable API")
Acked-by: Joe Lawrence <joe.lawrence@redhat.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Acked-by: Miroslav Benes <mbenes@suse.cz>
[nstange@suse.de: Apply patch to backport in kernel/kgraft_shadow.c.]
Signed-off-by: Nicolai Stange <nstange@suse.de>
---
 kernel/kgraft_shadow.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/kgraft_shadow.c
+++ b/kernel/kgraft_shadow.c
@@ -113,7 +113,7 @@ void *kgr_shadow_get(void *obj, unsigned
 }
 EXPORT_SYMBOL_GPL(kgr_shadow_get);
 
-void *__kgr_shadow_get_or_alloc(void *obj, unsigned long id, void *data,
+static void *__kgr_shadow_get_or_alloc(void *obj, unsigned long id, void *data,
 		       size_t size, gfp_t gfp_flags, bool warn_on_exist)
 {
 	struct kgr_shadow *new_shadow;
